specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-02-01-preview/sites.json

```diff
@@ -0,0 +1,266 @@
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SiteScope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SitesModel"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Create or update IoT site": {
+            "$ref": "./examples/Sites/Put.json"
+          }
+        },
+        "tags": [
+          "Sites"
+        ],
+        "operationId": "Sites_CreateOrUpdate",
+        "description": "Create or update IoT site",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SiteScope"
+          },
+          {
+            "$ref": "#/parameters/SitesModel"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated IoT site",
+            "schema": {
+              "$ref": "#/definitions/SitesModel"
+            }
+          },
+          "201": {
+            "description": "Created IoT site",
+            "schema": {
+              "$ref": "#/definitions/SitesModel"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete IoT site": {
+            "$ref": "./examples/Sites/Delete.json"
+          }
+        },
+        "tags": [
+          "Sites"
+        ],
+        "operationId": "Sites_Delete",
+        "description": "Delete IoT site",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SiteScope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted IoT site"
+          },
+          "204": {
+            "description": "IoT site does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
```

Comment:
Is this long-running?  See previous comments on synchronous versus async operations

Reply:
It's synchronous and isn't long-running